request.js

```diff
@@ -149,7 +149,21 @@ Request.prototype.init = function (options) {
     options = {}
   }
   self.headers = self.headers ? copy(self.headers) : {}
-
+  
+  // bind default events sent via options
+  // additional postman feature starts
+  if (options.bindOn) {
```

Comment:
Needs a stricter type check.

Reply:
I don't see request library doing it anywhere